SECURITY.md

```diff
@@ -0,0 +1,25 @@
+# Security Policy
+
+## Supported Versions
+
+We do not support any old versions, all updates for bugs/features/security 
+will be released in next planned version.
+
+| Version      | Supported          |
+| ------------ | ------------------ |
+| LATEST only  | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+We have mail-list
+https://checkstyle.org/mailing-lists.html
+you can find admin emails in it.
+you can find my email in my commits, you can clone our repo and see it by "git log".
```

Comment:
"my" seems confusing when reading this page. I doubt users will look up who created this MD.

Why not point them to admins at https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L32 ? We could add our emails there.

Reply:
sorry for copy-paste from private conversation .... fixed.
Am not sure why but I still have old fashioned position to not print email in plain text.

I updated a bit wording, please take a look.